src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/handlers/presubmit/AppealSubmissionCreator.java

```diff
@@ -26,13 +27,16 @@
 public class AppealSubmissionCreator implements PreSubmitCallbackHandler<AsylumCase> {
 
     private final DocumentCreator<AsylumCase> appealSubmissionDocumentCreator;
+    private final DocumentCreator<AsylumCase> internalAppealSubmissionDocumentCreator;
     private final DocumentHandler documentHandler;
 
     public AppealSubmissionCreator(
         @Qualifier("appealSubmission") DocumentCreator<AsylumCase> appealSubmissionDocumentCreator,
+        @Qualifier("internalAppealSubmissionNonAda") DocumentCreator<AsylumCase> internalAppealSubmissionDocumentCreator,
```

Comment:
@ikirsanov could you rename the qualifier or the variable to either reflect non ADA or all internal Appeals?

Reply:
Renaimed to internalAppealSubmission classes, properties